src/Plug.vala

```diff
@@ -355,7 +357,7 @@ namespace Power {
             grid.attach (sleep_timeout_label, 0, 1, 1, 1);
             grid.attach (sleep_timeout, 1, 1, 1, 1);
 
-            if (!ac) {
+            if (!ac && backlight_detect () ){
```

Comment:
no space after the parenthesis, it should look like this:
`if (!ac && backlight_detect ()) {`